projects/goharbor/harbor/docker/linux/harbor-core/Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-base
```

Comment:
If we are using the standard base, which I would rather we didnt have to since it increases your CVE landscape due to a larger number of OS packages you dont even need, then you do not to do all the useradd stuff in the makefile.  Instead you can do it like you did here using shaddow-utils.  I would probably remove shadow utils in the same layer after creating the user/group just to avoid it being in the final image.